lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,21 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        case app_name
+        when '.'
+          ::File.basename(Dir.getwd)
+        when '/'
```

Comment:
@mengqing @joneslee85 To cover that case, I would prefer to have a `--path` option. This makes both the intent of the CLI and the internals more clear and more protected from bugs.